unity-environment/Assets/ML-Agents/Examples/Banana/Scripts/BananaArea.cs

```diff
@@ -11,12 +11,13 @@ public class BananaArea : Area
     public int numBananas;
     public int numBadBananas;
     public bool respawnBananas;
-
+    public float range;
     // Use this for initialization
     void Start()
     {
         aca = GameObject.Find("Academy").GetComponent<BananaAcademy>();
         aca.listArea.Add(this);
+        range = 20f;
```

Comment:
comment on the 20f, and ideally make range a const since it doesn't seem to be modified anywhere (?)